src/components/SelectLanguage.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+function SelectLanguage() {
+  const [language, setLanguage] = useState('en');
+  const onSelect = (e) => {
+    e.preventDefault();
+    setLanguage(e.target.value);
+  };
+  return (
+    <select
```

Comment:
Hey Chris, I think it will be a good idea to use **spans** instead of select to change languages, I don't have any idea on how to change the value of default value of options after changing language.

Reply:
ok I will work on it